src/dx/comms/resample.py

```diff
@@ -25,6 +25,6 @@ def handle_resample_comm(msg):
     if not data:
         return
 
-    if "display_id" in data and "filters" in data:
-        msg = DEXResampleMessage.parse_obj(data)
-        handle_resample(msg)
+    logger.debug(f"handling resample {msg=}")
```

Comment:
failure to parse into a DEXResampleMessage should raise an error that's captured by the frontend / originator of the comm_msg